gamemodes/terrortown/gamemode/shared/lang/italian.lua

```diff
@@ -0,0 +1,1273 @@
+-- Translated by Alessandro Levante (AL24)
+-- Tradotto da Alessandro Levante (AL24)
+
+-- Stinghe in italiano
+-- Italian language strings
+
+local L = LANG.CreateLanguage("Italian")

```

Comment:
you might want to change this to your italian name, because for example it is "Deutsch" for German